docs/developer-docs/integrations/sns/managing/sns-asset-canister.md

```diff
@@ -22,49 +29,23 @@ The general overview of deploying an asset canister during an SNS launch is as f
     - The user or developer creating the SNS should remove their own personal permissions. 
 - Lastly, the SNS's function should be registered to commit the configuration.
 
-To deploy an asset canister, first you will need an `assets.wasm.gz` file and a `candid/assets.did` file. Examples of these files can be downloaded with the commands:
+To deploy an asset canister, any canister in the `dfx.json` file can be set as `"type": "assets"`, and dfx will automatically generate the required files, such as an assets.wasm.gz file and a candid/assets.did file once the canister has been built. 
 
-```
-curl -L "https://github.com/dfinity/sdk/raw/${DX_COMMIT}/src/distributed/assetstorage.wasm.gz" -o assets.wasm.gz
-curl -L "https://raw.githubusercontent.com/dfinity/sdk/${DX_COMMIT}/src/distributed/assetstorage.did" -o candid/assets.did
-```
-
-Next, you will need to configure your `dfx.json` file to specify your asset canister. An example `dfx.json` file can be found below:
+An example of configuring an asset canister within the `dfx.json` file can be found below:
 
 ```
 {
   "canisters": {
-    "nns-dapp": {
-      "type": "custom",
-      "candid": "",
-      "wasm": "nns-dapp.wasm"
-    },
-    "internet_identity": {
-      "type": "custom",
-      "candid": "",
-      "wasm": "internet_identity.wasm"
-    },
-    "sns_aggregator": {
-      "type": "custom",
-      "candid": "candid/sns_aggregator.did",
-      "wasm": "sns_aggregator.wasm"
-    },
-    "test": {
-      "candid": "candid/test.did",
-      "type": "rust",
-      "package": "test"
-    },
     "assets": {
```

Comment:
In addition to setting the "type" field, I'd also drop the remaining fields and use something like this:
```
    "assets": {
      "source": [],
      "type": "assets"
    }
```